dev/CsvBulkLoader.php

```diff
@@ -47,8 +56,99 @@ public function preview($filepath) {
 	/**
 	 * @param string $filepath
 	 * @param boolean $preview
+	 *
+	 * @return null|BulkLoader_Result
 	 */
 	protected function processAll($filepath, $preview = false) {
+		$files = $this->splitFile($filepath);
+
+		$result = null;
+
+		foreach ($files as $file) {
+			$next = $this->processChunk($file);
+
+			if ($result instanceof BulkLoader_Result) {
+				$result->merge($next);
+			} else {
+				$result = $next;
+			}
+
+			@unlink($file);
```

Comment:
Maybe wrap this whole loop in a try/catch as well with the unlink in the catch? `tinfoilhat`